### DISCUSSION TITLE
Separate tar Trap/Flare

### DISCUSSION THREAD

**Discussion by TheTylerOlson** [Hekili/hekili#788] [Q&A] [answered]
Is there anyway to separate Tar Trap/Flare (for Soulforge Embers) from my Hunter rotation so I can manage that separately from my normal rotation?

**Comment by Hekili** [comment, marked as answer]
You could set those abilities to the "interrupts" toggle and set your interrupts to show separately (Toggles screen), then they would show up in your Interrupts display instead of your Primary/AOE displays.
